test/reshape.jl

```diff
@@ -36,8 +36,8 @@ const ≅ = isequal
                    Value = Union{String, Missing}["12 g", "Red", "18 g", "Grey"])
     levels!(df[!, 1], ["XXX", "Bob", "Batman"])
     levels!(df[!, 2], ["YYY", "Color", "Mass"])
-    df2 = unstack(df, :Fish, :Key, :Value, renamecols=x->string("_", uppercase(x), "_"))
-    df3 = unstack(df, :Key, :Value, renamecols=x->string("_", uppercase(x), "_"))
+    df2 = unstack(df, :Fish, :Key, :Value, renamecols=x->string("_", uppercase(string(x)), "_"))
+    df3 = unstack(df, :Key, :Value, renamecols=x->string("_", uppercase(string(x)), "_"))
```

Comment:
I don't understand this.

Reply:
@nalimilan 😄 has deprecated string operations on `CategoricalValue{String}`, so we have to unwrap it before we `uppercase` it.